docs/asset-store/06-03-clusterasset.md

```diff
@@ -86,43 +65,68 @@ status:
+| **spec.source.mutationwebhookservice.name** | Yes | Provides the name of the mutation webhook service. |
+| **spec.source.mutationwebhookservice.namespace** | Yes | Provides the Namespace in which the service is available. |
+| **spec.source.mutationwebhookservice.endpoint** | No | Specifies the endpoint to which the service sends calls. |
+| **spec.source.mutationwebhookservice.parameters** | No | Provides detailed parameters specific for a given mutation service and its functionality. |
+| **spec.source.mutationwebhookservice.filter** | No | Specifies the regex pattern used to select files sent to the service. |
+| **spec.source.metadatawebhookservice** | No | Provides specification of the metadata webhook services. |
+| **spec.source.metadatawebhookservice.name** | Yes | Provides the name of the metadata webhook service. |
+| **spec.source.metadatawebhookservice.namespace** | Yes  | Provides the Namespace in which the service is available. |
+| **spec.source.metadatawebhookservice.endpoint** | No | Specifies the endpoint to which the service sends calls. |
+| **spec.source.metadatawebhookservice.filter** | No | Specifies the regex pattern used to select files sent to the service. |
+| **spec.bucketref.name** | Yes | Provides the name of the bucket for storing the asset. |
+| **status.phase** | Not applicable | The ClusterAsset Controller adds it to the ClusterAsset CR. It describes the status of processing the ClusterAsset CR by the ClusterAsset Controller. It can be `Ready`, `Failed`, or `Pending`. |
+| **status.reason** | Not applicable | Provides the reason why the ClusterAsset CR processing failed or is pending. See the [**Reasons**](#status-reasons) section for the full list of possible status reasons and their descriptions.  |
+| **status.message** | Not applicable | Describes a human-readable message on the CR processing progress, success, or failure. |
+| **status.lastheartbeattime** | Not applicable | Provides the last time when the ClusterAsset Controller processed the ClusterAsset CR. |
+| **status.observedGeneration** | Not applicable | Specifies the most recent generation that the ClusterAsset Controller observes. |
+| **status.assetref** | Not applicable | Provides details on the location of the assets stored in the bucket.   |
+| **status.assetref.assets** | Not applicable | Provides the relative path to the given asset in the storage bucket. |
+| **status.assetref.baseurl** | Not applicable | Specifies the absolute path to the location of the assets in the storage bucket.   |
 
 
 > **NOTE:** The ClusterAsset Controller automatically adds all parameters marked as **Not applicable** to the ClusterAsset CR.
 
+### Status reasons
+
+Processing of a ClusterAsset CR can succeed, continue, or fail for one of these reasons:
+
+| Reason | Phase | Description |
+| --------- | ------------- | ----------- |
+| `Pulled` | `Ready` | The Asset Controller pulled the asset content for processing. |
```

Comment:
Same content as for Asset CR, also maybe we can move this section to a separate doc, because right now for both types it always will be the same.

Reply:
I will think about it. I was already considering it but decided that since it is an integral part of the CRD, I will keep it that way - but true, there is no differentiation in the description between ClusterAssets and Assets... @derberg what do you think?